schema-source/RSxxSub.schema.yaml

```diff
@@ -0,0 +1,224 @@
+Schema:
+  Object Type: "Meta"
+  References:
+    - "ASHRAE205"
+  Title: "Fenestration - Glazing Substrate"
+  Description: "Schema for ASHRAE 205 annex RSxxSub: Fenestration - Glazing Substrate" 
+  Version: "2.0.0"
+  Root Data Group: "RSxxSub"
+
+# Data Groups
+RSxxSub:
+  Object Type: "Data Group"
+  Data Elements:
+    metadata:
+      Description: "Metadata data group"
+      Data Type: "{Metadata}"
+      Constraints: "schema=RSxxSub"
+      Required: True
+    description:
+      Description: "Data group describing product and rating information"
+      Data Type: "{Description}"
+    performance:
+      Description: "Data group containing performance information"
+      Data Type: "{Performance}"
+      Required: True
+
+Description:
+  Object Type: "Data Group"
+  Data Elements:
+    product_information:
+      Description: "Data group describing product information"
+      Data Type: "{ProductInformation}"
+
+ProductInformation:
+  Object Type: "Data Group"
+  Data Elements:
+    manufacturer:
+      Description: "Manufacturer name"
+      Data Type: "String"
+    model_number:
+      Description: "Model number"
+      Data Type: "Pattern"
+      Notes: "Pattern shall match all model numbers that can be represented by the representation"
+
+Performance:
+  Object Type: "Data Group"
+  Data Elements:
+    thickness:
+      Description: "The total thickness of the glazing."
+      Data Type: "Numeric"
+      Units: "m"
+      Required: True
+    thermal_conductivity:
+      Description: "Overall thermal conductivity of the glazing."
+      Data Type: "Numeric"
+      Units: "W/m-K"
+      Required: True
+    front_side_infrared_emissivity:
+      Description: "Front Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: True
+      Constraints: [">=0.0", "<=1.0"]
+    back_side_infrared_emissivity:
```

Comment:
Where do you define "front" and "back"? Panes can be installed "flipped".

Reply:
I will add a definition for front and back.